downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

```diff
@@ -88,6 +88,12 @@ To view bug fixes, see the [GitHub milestone for 2201.4.0 (Swan Lake)](https://g
 ### Improvements
 
 #### OpenAPI Tool
+- Added mapping for the `additionalProperty` property in the OpenAPI object schema. With this support, generated Ballerina records can be open records or closed records according to the `additionalProperty` property details.
+  Thereby, some of your existing generated Ballerina records may change into close records after regenerating using Ballerina 2201.4.0 (Swan Lake) version.
+- Improved OpenAPI schema `nullable:true` property support by providing generated record field with default of value nil, instead of making field both nilable and optional.
```

Comment:
```suggestion
- Improved the OpenAPI schema `nullable:true` property support by providing a generated record field with `nil` as the default value instead of making the field both nilable and optional.
```
What is the correct word `nilable` or `nullable`?